benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/BenchFlowExperimentConfigurationYamlProtocol.scala

```diff
@@ -16,6 +16,10 @@ import scala.util.Try
  */
 object BenchFlowExperimentConfigurationYamlProtocol extends DefaultYamlProtocol {
 
+  // default value since deserializing from a test definition it might not be defined
+  // before use it will be overridden
+  val defaultUsers = 0
```

Comment:
Set `-1` and open an issue to explain how these values are handled later on when generating the actual experiment.